mix.lock

```diff
@@ -62,7 +62,7 @@
   "recon": {:hex, :recon, "2.5.6", "9052588e83bfedfd9b72e1034532aee2a5369d9d9343b61aeb7fbce761010741", [:mix, :rebar3], [], "hexpm", "96c6799792d735cc0f0fd0f86267e9d351e63339cbe03df9d162010cefc26bb0"},
   "remote_ip": {:hex, :remote_ip, "1.2.0", "fb078e12a44414f4cef5a75963c33008fe169b806572ccd17257c208a7bc760f", [:mix], [{:combine, "~> 0.10", [hex: :combine, repo: "hexpm", optional: false]}, {:plug, "~> 1.14", [hex: :plug, repo: "hexpm", optional: false]}], "hexpm", "2ff91de19c48149ce19ed230a81d377186e4412552a597d6a5137373e5877cb7"},
   "retry": {:hex, :retry, "0.18.0", "dc58ebe22c95aa00bc2459f9e0c5400e6005541cf8539925af0aa027dc860543", [:mix], [], "hexpm", "9483959cc7bf69c9e576d9dfb2b678b71c045d3e6f39ab7c9aa1489df4492d73"},
-  "screens_config": {:git, "https://github.com/mbta/screens-config-lib.git", "5b56f8dbf03e4c1f08d451272980e15a5bcf3b92", []},
+  "screens_config": {:git, "https://github.com/mbta/screens-config-lib.git", "2f41fff8f0f22c6d96bfc5616a945982e9205e6b", []},
```

Comment:
just pointing out, this commit doesn't include your latest commit to screens-config-lib. Since that's just essentially a spec change, might as well upgrade it here

Reply:
That is covered by #2577, which is based off this branch!